boards/arm/mimxrt685_evk/flash_flexspi_nor_lut.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "flash_mcux_flexspi_nor_lut_index.h"
+
+static const uint32_t flash_flexspi_nor_lut[][4] = {
+	[READ_ID] = {
+		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_SDR,       kFLEXSPI_1PAD, SPI_NOR_CMD_RDID,
+				kFLEXSPI_Command_READ_SDR,  kFLEXSPI_1PAD, 0x04),
+	},
+
+	[READ_STATUS_REG] = {
+		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_SDR, kFLEXSPI_8PAD, 0x05,
+				kFLEXSPI_Command_SDR, kFLEXSPI_8PAD, 0xFA),
+		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_RADDR_SDR, kFLEXSPI_8PAD, 0x20,
+				kFLEXSPI_Command_DUMMY_SDR, kFLEXSPI_8PAD, 0x14),
+		FLEXSPI_LUT_SEQ(kFLEXSPI_Command_READ_SDR, kFLEXSPI_8PAD, 0x04,
+				kFLEXSPI_Command_STOP, kFLEXSPI_1PAD, 0x0),
```

Comment:
Please fix the whitespace to follow the convention in the existing LUT

Reply:
Done.